### PR TITLE
Goreportcard

### PR DESCRIPTION
**Pull request by andrecronje**
Modifying for results found in;

https://goreportcard.com/report/github.com/Fantom-foundation/go-lachesis
https://golangci.com/r/github.com/Fantom-foundation/go-lachesis

Roughly 50% complete, will continue working on this PR

### CODE REVIEW

File: cmd/lachesis/commands/run_lachesis.go [hunk 1]
```diff
@@ -1,8 +1,10 @@
+package commands
```
Comment (Maxime2) [inline cmd/lachesis/commands/run_lachesis.go]: Build tags (comment lines starting with // +build ) must appear near the top of the file, preceded only by blank lines and other line comments. (see https://golang.org/pkg/go/build/ ).
Thus  package command must be put back down under build tag.

Comment (andrecronje) [reply cmd/lachesis/commands/run_lachesis.go]: Updated, thanks for catching!

File: src/proxy/internal/grpc.pb.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ const _ = proto.ProtoPackageIsVersion2 // please upgrade the proto package
 type ToServer struct {
 	// Types that are valid to be assigned to Event:
 	//	*ToServer_Tx_
-	//	*ToServer_Answer_
```
Comment (devintegral) [inline src/proxy/internal/grpc.pb.go]: Do not edit this file: code generated by protoc-gen-go from grpc.proto

File: src/proxy/grpc_app.go [hunk 3]
```diff
@@ -33,16 +35,16 @@ type GrpcAppProxy struct {
 	server   *grpc.Server
 
 	timeout      time.Duration
-	new_clients  chan ClientStream
-	askings      map[xid.ID]chan *internal.ToServer_Answer
-	askings_sync sync.RWMutex
+	newClients  chan ClientStream
+	askings      map[xid.ID]chan *internal.ToServerAnswer
```
Comment (devintegral) [inline src/proxy/grpc_app.go]: Do not rename internal-package's entities: they are generated by protoc-gen-go from grpc.proto. The same below

**Pull request closed without merging**